src/java/org/apache/cassandra/exceptions/TombstoneAbortException.java

```diff
@@ -23,14 +23,14 @@
 import org.apache.cassandra.db.ConsistencyLevel;
 import org.apache.cassandra.locator.InetAddressAndPort;
 
-import static org.apache.cassandra.service.reads.ReadCallback.tombstoneAbortMessage;
+import static org.apache.cassandra.service.reads.trackwarnings.WarningsSnapshot.tombstoneAbortMessage;
```

Comment:
nit: alternate package naming

```suggestion
import static org.apache.cassandra.service.reads.warnings.WarningsSnapshot.tombstoneAbortMessage;
```